sklearn/ensemble/weight_boosting.py

```diff
@@ -33,6 +33,7 @@
 from ..externals.six.moves import zip
 from ..externals.six.moves import xrange as range
 from .forest import BaseForest
+from ..feature_selection.from_model import _get_feature_importances
```

Comment:
If it is a shared utility, it should live in sklearn.utils

Reply:
The implementation currently mentions SelectFromModel and should not

Reply:
I see two solutions:
1. Move `_get_feature_importances` to `sklearn.utils.__init__.py` or create a new file (`feature_importances.py`) within `utils` and move it there; import this function into all those modules it's being used (2 places)
2. Without reusing this function, duplicate its functionality with-in `weight_boosting.py`; if the above step is unnecessary. 

@jnothman Which one should I choose?